scss/modules/_mixins.scss

```diff
@@ -0,0 +1,130 @@
+// ----------------------------------------------------
+// ----- MIXINS -----
+// ----------------------------------------------------
+
+  // --- Media Queries - Breakpoints ---
+  @mixin screen($size) {
```

Comment:
Nie wiem skąd wzięłaś taką definicję RWD ale jest to zdecydowanie zbyt skomplikowane. Wykonujesz za dużo operacji i przez to kod jest mniej wydajny.